crates/router/src/core/payments.rs

```diff
@@ -2263,6 +2263,7 @@ where
     pub authorizations: Vec<diesel_models::authorization::Authorization>,
     pub authentication: Option<storage::Authentication>,
     pub frm_metadata: Option<serde_json::Value>,
+    pub recurring_details: Option<String>,
```

Comment:
It's better if this is the enum itself. With a `String`, we won't know later on in the flow whether it's a recurring mandate or a recurring MIT payment.